pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -70,6 +70,7 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   REFRESH_FAILURES("segments", false),
   UNTAR_FAILURES("segments", false),
   SEGMENT_STREAMED_DOWNLOAD_UNTAR_FAILURES("segments", false),
+  SEGMENT_TOTAL_STREAMED_DOWNLOAD_UNTAR_FAILURES("segments", false),
```

Comment:
Can you please add metrics description as a part of this definition? You can see some examples in [`ServerTimer`](https://github.com/apache/pinot/blob/master/pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerTimer.java#L29)

Reply:
Done in commit 84c64c1f925b9fabe62f51c9ca2a466414fff34d